HeterogeneousCore/CUDAServices/interface/CUDAService.h

```diff
@@ -52,7 +52,7 @@ class CUDAService {
   bool enabled() const { return enabled_; }
   // To be used in stream context when an edm::Stream is available
   bool enabled(edm::StreamID streamId) const { return enabled(static_cast<unsigned int>(streamId)); }
-  bool enabled(unsigned int streamId) const { return enabled_ && (numberOfStreamsTotal_ == 0 || streamId < numberOfStreamsTotal_); } // to make testing easier
+  bool enabled(unsigned int streamId) const { return enabled_; } // to make testing easier
```

Comment:
can we remove `bool enabled(edm::StreamID streamId)` and `bool enabled(unsigned int streamId)` ?

Reply:
Sure, they don't really make sense anymore.